bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/TargetEnvironmentDiff.java

```diff
@@ -62,22 +55,29 @@ private static TargetEnvironmentDiff primaryGuideGroupExtraction(TargetEnvironme
     private final Collection<SPTarget> removedTargets;
     private final Collection<SPTarget> addedTargets;
 
-    private TargetEnvironmentDiff(TargetEnvironment oldEnv, TargetEnvironment newEnv,
-                                  ImList<SPTarget> oldTargets, ImList<SPTarget> newTargets) {
+    private final Collection<SPCoordinates> removedCoordinates;
+    private final Collection<SPCoordinates> addedCoordinates;
+
+    // Return an ImList<T> containing the elements of list1 \ list2.
```

Comment:
Ahhh... it's a slash and not a vertical bar: it just looks ambiguous in italics. When I learned set theory, we wrote set difference as `S \ T`, but since this isn't clear, I'll change to the minus symbol.